File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
       if(nums.Length<2)
        return false;
        Dictionary<int,int> valueSet= new Dictionary<int,int>();
        for(int i=0; i<nums.Length;i++){
            if(valueSet.ContainsKey(nums[i]))
                return true;
            valueSet[nums[i]]=i;
        }
        return false;
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int n = nums.length-1;
        int sum=0;
        int maxSum=nums[0];
        for (int i=0; i<= n ; i++){
            sum= nums[i];
            if(sum>maxSum){
                maxSum=sum;
            }
            for(int j=i+1; j<=n;j++){
                sum += nums[j];
                if(sum>maxSum){
                    maxSum=sum;
                }
            }
        }
        return maxSum;
    }
}
